.circleci/config.yml

```diff
@@ -76,6 +78,22 @@ jobs:
           paths:
             - ~/.cache/yarn
             - node_modules
+
+  schematics-core-copy:
```

Comment:
I would change this because this seems like it copies the file. 
Something as `check-schematics-core-copied`?

Reply:
Exactly, it seems more easier to understand, thank you.